spmd/tensor/dispatch.py

```diff
@@ -111,20 +111,12 @@ def _reshape_alias(
 ] = {torch.ops.aten._reshape_alias.default: _reshape_alias}
 
 
-def operator_dispatch(
+def prepare_inputs(
```

Comment:
could we rename this to sth like `propagate_input_sharding`? as it also calling into sharding prop and does not only prepare the inputs

Reply:
Sure, let me update the name